udfs/tests/udf_test.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import listdir
+from os import path
+import re
+import unittest
+
+from google.api_core.exceptions import GoogleAPICallError
+from google.cloud.bigquery import QueryJobConfig
+from google.cloud.bigquery.table import _EmptyRowIterator
+from yaml import SafeLoader
+from yaml import load
+
+
+class UDFTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Client object is initialized in child class
```

Comment:
Is this then an abstract base class? if so UDFTest should inherit from [ABC](https://docs.python.org/3/library/abc.html)

Reply:
Disregard this comment if refactoring to be a single concrete test class.